### ISSUE TITLE
opam show could be more consistent with newlines

### ISSUE THREAD

**Issue by avsm** [ocaml/opam#3415]
This is frustratingly hard to parse from a shell script :)

```
$ opam show -f depends: sexplib0 
"ocaml" {>= "4.04.1"} "jbuilder" {build & >= "1.0+beta18.1"}
$ opam show -f depends: mirage 
"ocaml" {>= "4.04.2"}
"ocamlbuild"
"ocamlfind"
"jbuilder" {build & >= "1.0+beta10"}
"ipaddr" {>= "2.6.0"}
"functoria" {>= "2.2.0"}
"bos"
"astring"
"logs"
"mirage-runtime" {>= "3.0.0"}
```

If `opam show` would just make every list entry into a newline, it would be so much simpler to parse with line-oriented tools in a shell.  Using `--normalise` adds brackets and makes it harder to parse.

**Comment by avsm** [comment]
I'm also noticing that the output of `opam show --normalise` can't always be parsed by `OpamParser.value_of_string`.  If there is an empty string, a `Parsing_error` is raised.

**Comment by dbuenzli** [comment]
Also would it be possible to only report what is in the file when this is performed on an opam file; it seems that `opam` will invent things in some cases (here on an `opam` file that has no `version:` field):

```
> opam show ./opam -f version:,homepage: --normalise --no-lint
version:  "~dev"
homepage: ["bla" "bli" "blo"]
```

**Comment by rjbou** [comment]
It have its origins in [opam-file-format](https://github.com/ocaml/opam-file-format/blob/master/src/opamPrinter.ml#L90). If we change the printer at this stage, it will change opam printing for all tools, even camelus. Meanwhile, it is possible to pin for the CI a version of `opam-file-format` that "disable" that formatting.
